task/utils.go

```diff
@@ -0,0 +1,86 @@
+package task
+
+import (
+	"fmt"
+
+	"github.com/aos-dev/go-storage/v3/types"
+)
+
+func calculatePartSize(o *types.Object) (int64, error) {
```

Comment:
Logic is incorrect.

The object to be uploaded can't be the multipart object here.